src/taoensso/carmine/message_queue.clj

```diff
@@ -106,24 +106,25 @@
                            perform a de-duplication check. If unspecified, a
                            unique id will be auto-generated.
     * allow-requeue?     - When true, allow buffered escrow-requeue for a
-                           message in the :locked or :done-with-backoff state."
-  ;; TODO Option to enqueue something with an init backoff?
+                           message in the :locked or :done-with-backoff state.
+    * initial-backoff-ms - Initial backoff in millis."
   (let [script (enc/have (enc/slurp-resource "taoensso/carmine/lua/mq/enqueue.lua"))]
-    (fn [qname message & [unique-message-id allow-requeue?]]
+    (fn [qname message & {:keys [unique-message-id allow-requeue? initial-backoff-ms]}]
```

Comment:
Would request `[{:keys [unique-message-id allow-requeue? initial-backoff-ms]}]` as the optional part here (note the enclosing vector).

This'll mean calling `enqueue` like
`(enqueue qname message {:initial-backoff-ms 1000})` rather than
`(enqueue qname message :initial-backoff-ms 1000)`.

The former is easier to manipulate programmatically for composition, libraries, etc.

Reply:
Done. Just pushed this change and rebased the PR.